### DISCUSSION TITLE
[Bug] Cant install on MacOS Intel

### DISCUSSION THREAD

**Discussion by alxbouchard** [coqui-ai/TTS#2896] [General Q&A]
### Describe the bug

Cant install on MacOS Intel

### To Reproduce

`           INFO: clang: numpy/core/src/multiarray/flagsobject.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            numpy/core/src/multiarray/einsum.c.src:408:32: warning: unknown warning group '-Wmaybe-uninitialized', ignored [-Wunknown-warning-option]
            #pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
                                           ^
            1 warning generated.
            1 warning generated.
            INFO: clang: numpy/core/src/multiarray/item_selection.c
            1 warning generated.
            INFO: clang: numpy/core/src/multiarray/dlpack.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            INFO: clang: numpy/core/src/multiarray/ctors.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            INFO: clang: build/src.macosx-10.9-universal2-3.11/numpy/core/src/multiarray/einsum_sumprod.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            INFO: clang: numpy/core/src/multiarray/getset.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            1 warning generated.
            INFO: clang: numpy/core/src/multiarray/datetime_busday.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            INFO: clang: build/src.macosx-10.9-universal2-3.11/numpy/core/src/multiarray/lowlevel_strided_loops.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            1 warning generated.
            INFO: clang: numpy/core/src/multiarray/hashdescr.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            1 warning generated.
            INFO: clang: numpy/core/src/multiarray/multiarraymodule.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            INFO: clang: numpy/core/src/multiarray/nditer_constr.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            INFO: clang: numpy/core/src/multiarray/refcount.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            INFO: clang: numpy/core/src/multiarray/sequence.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            INFO: clang: numpy/core/src/multiarray/scalarapi.c
            INFO: clang: numpy/core/src/multiarray/shape.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            numpy/core/src/multiarray/scalarapi.c:773:37: warning: 'ob_shash' is deprecated [-Wdeprecated-declarations]
                        ((PyBytesObject *)obj)->ob_shash = -1;
                                                ^
            /Library/Frameworks/Python.framework/Versions/3.11/include/python3.11/cpython/bytesobject.h:7:5: note: 'ob_shash' has been explicitly marked deprecated here
                Py_DEPRECATED(3.11) Py_hash_t ob_shash;
                ^
            /Library/Frameworks/Python.framework/Versions/3.11/include/python3.11/pyport.h:336:54: note: expanded from macro 'Py_DEPRECATED'
            #define Py_DEPRECATED(VERSION_UNUSED) __attribute__((__deprecated__))
                                                                 ^
            INFO: clang: numpy/core/src/multiarray/iterators.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            2 warnings generated.
            1 warning generated.
            numpy/core/src/multiarray/scalarapi.c:773:37: warning: 'ob_shash' is deprecated [-Wdeprecated-declarations]
                        ((PyBytesObject *)obj)->ob_shash = -1;
                                                ^
            /Library/Frameworks/Python.framework/Versions/3.11/include/python3.11/cpython/bytesobject.h:7:5: note: 'ob_shash' has been explicitly marked deprecated here
                Py_DEPRECATED(3.11) Py_hash_t ob_shash;
                ^
            /Library/Frameworks/Python.framework/Versions/3.11/include/python3.11/pyport.h:336:54: note: expanded from macro 'Py_DEPRECATED'
            #define Py_DEPRECATED(VERSION_UNUSED) __attribute__((__deprecated__))
                                                                 ^
            1 warning generated.
            1 warning generated.
            INFO: clang: build/src.macosx-10.9-universal2-3.11/numpy/core/src/multiarray/scalartypes.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            INFO: clang: numpy/core/src/multiarray/temp_elide.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            1 warning generated.
            1 warning generated.
            INFO: clang: numpy/core/src/multiarray/typeinfo.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            INFO: clang: numpy/core/src/multiarray/usertypes.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            INFO: clang: numpy/core/src/multiarray/legacy_dtype_implementation.c
            INFO: clang: numpy/core/src/multiarray/nditer_pywrap.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            INFO: clang: numpy/core/src/multiarray/vdot.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            1 warning generated.
            1 warning generated.
            INFO: clang: build/src.macosx-10.9-universal2-3.11/numpy/core/src/npysort/timsort.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            INFO: clang: build/src.macosx-10.9-universal2-3.11/numpy/core/src/npysort/quicksort.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            1 warning generated.
            INFO: clang: numpy/core/src/multiarray/number.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            INFO: clang: build/src.macosx-10.9-universal2-3.11/numpy/core/src/multiarray/nditer_templ.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            INFO: clang: numpy/core/src/multiarray/strfuncs.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            INFO: clang: build/src.macosx-10.9-universal2-3.11/numpy/core/src/npysort/binsearch.c
            1 warning generated.
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            INFO: clang: build/src.macosx-10.9-universal2-3.11/numpy/core/src/umath/loops.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            1 warning generated.
            1 warning generated.
            INFO: clang: numpy/core/src/umath/umathmodule.c
            INFO: clang: build/src.macosx-10.9-universal2-3.11/numpy/core/src/npysort/mergesort.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            INFO: clang: numpy/core/src/multiarray/nditer_api.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            INFO: clang: numpy/core/src/umath/reduction.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            INFO: clang: numpy/core/src/multiarray/experimental_public_dtype_api.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            1 warning generated.
            INFO: clang: numpy/core/src/umath/legacy_array_method.c
            INFO: clang: build/src.macosx-10.9-universal2-3.11/numpy/core/src/umath/scalarmath.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            1 warning generated.
            1 warning generated.
            INFO: clang: numpy/core/src/umath/ufunc_object.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            1 warning generated.
            INFO: clang: numpy/core/src/umath/_scaled_float_dtype.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            INFO: clang: numpy/core/src/common/array_assign.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            INFO: clang: numpy/core/src/common/mem_overlap.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            1 warning generated.
            INFO: clang: numpy/core/src/common/npy_argparse.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            INFO: clang: numpy/core/src/common/npy_hashtable.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            INFO: clang: numpy/core/src/common/ucsnarrow.c
            1 warning generated.
            1 warning generated.
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            INFO: clang: numpy/core/src/common/npy_longdouble.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            INFO: clang: numpy/core/src/common/ufunc_override.c
            1 warning generated.
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            INFO: clang: build/src.macosx-10.9-universal2-3.11/numpy/core/src/common/npy_cpu_features.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            INFO: clang: numpy/core/src/common/numpyos.c
            INFO: clang: numpy/core/src/multiarray/array_coercion.c
            numpy/core/src/common/npy_cpu_features.c.src:125:1: warning: unused function 'npy__cpu_baseline_fid' [-Wunused-function]
            npy__cpu_baseline_fid(const char *feature)
            ^
            numpy/core/src/common/npy_cpu_features.c.src:138:1: warning: unused function 'npy__cpu_dispatch_fid' [-Wunused-function]
            npy__cpu_dispatch_fid(const char *feature)
            ^
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            3 warnings generated.
            numpy/core/src/common/npy_cpu_features.c.src:125:1: warning: unused function 'npy__cpu_baseline_fid' [-Wunused-function]
            npy__cpu_baseline_fid(const char *feature)
            ^
            numpy/core/src/common/npy_cpu_features.c.src:138:1: warning: unused function 'npy__cpu_dispatch_fid' [-Wunused-function]
            npy__cpu_dispatch_fid(const char *feature)
            ^
            2 warnings generated.
            INFO: clang: numpy/core/src/common/cblasfuncs.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            1 warning generated.
            1 warning generated.
            INFO: clang: numpy/core/src/multiarray/array_method.c
            INFO: clang: numpy/core/src/umath/extobj.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            INFO: clang: numpy/core/src/common/python_xerbla.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            1 warning generated.
            1 warning generated.
            INFO: clang: numpy/core/src/umath/ufunc_type_resolution.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            INFO: clang: build/src.macosx-10.9-universal2-3.11/numpy/core/src/npysort/heapsort.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            INFO: clang: numpy/core/src/umath/override.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            1 warning generated.
            INFO: clang: build/src.macosx-10.9-universal2-3.11/numpy/core/src/npysort/selection.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            INFO: clang: numpy/core/src/multiarray/mapping.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            INFO: clang: numpy/core/src/multiarray/methods.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            1 warning generated.
            INFO: clang: build/src.macosx-10.9-universal2-3.11/numpy/core/src/umath/matmul.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            INFO: clang: numpy/core/src/umath/dispatching.c
            warning: overriding currently unsupported use of floating point exceptions on this target [-Wunsupported-floating-point-opt]
            1 warning generated.
            error: Command "clang -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -arch arm64 -arch x86_64 -g -ftrapping-math -DNPY_INTERNAL_BUILD=1 -DHAVE_NPY_CONFIG_H=1 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE=1 -D_LARGEFILE64_SOURCE=1 -DNO_ATLAS_INFO=3 -DHAVE_CBLAS -Ibuild/src.macosx-10.9-universal2-3.11/numpy/core/src/common -Ibuild/src.macosx-10.9-universal2-3.11/numpy/core/src/umath -Inumpy/core/include -Ibuild/src.macosx-10.9-universal2-3.11/numpy/core/include/numpy -Ibuild/src.macosx-10.9-universal2-3.11/numpy/distutils/include -Inumpy/core/src/common -Inumpy/core/src -Inumpy/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inumpy/core/src/umath -Inumpy/core/src/npysort -Inumpy/core/src/_simd -I/Users/zap/myenv/include -I/Library/Frameworks/Python.framework/Versions/3.11/include/python3.11 -Ibuild/src.macosx-10.9-universal2-3.11/numpy/core/src/common -Ibuild/src.macosx-10.9-universal2-3.11/numpy/core/src/npymath -c numpy/core/src/multiarray/dragon4.c -o build/temp.macosx-10.9-universal2-3.11/numpy/core/src/multiarray/dragon4.o -MMD -MF build/temp.macosx-10.9-universal2-3.11/numpy/core/src/multiarray/dragon4.o.d -msse3 -I/System/Library/Frameworks/vecLib.framework/Headers" failed with exit status 1
            INFO:
            ########### EXT COMPILER OPTIMIZATION ###########
            INFO: Platform      :
              Architecture: unsupported
              Compiler    : clang
      
            CPU baseline  :
              Requested   : optimization disabled
              Enabled     : none
              Flags       : none
              Extra checks: none
              Requested   : optimization disabled
      
            CPU dispatch  :
              Enabled     : none
              Generated   : none
            INFO: CCompilerOpt.cache_flush[817] : write cache to path -> /private/var/folders/20/0qr8ky2558d47l3llvpglfz00000gn/T/pip-install-k3xzvyzv/numpy_e1d80e0ac53a4ad0b7fecb1268e59842/build/temp.macosx-10.9-universal2-3.11/ccompiler_opt_cache_ext.py
            INFO:
            ########### CLIB COMPILER OPTIMIZATION ###########
            INFO: Platform      :
              Architecture: unsupported
              Compiler    : clang
      
            CPU baseline  :
              Requested   : optimization disabled
              Enabled     : none
              Flags       : none
              Extra checks: none
              Requested   : optimization disabled
      
            CPU dispatch  :
              Enabled     : none
              Generated   : none
            INFO: CCompilerOpt.cache_flush[817] : write cache to path -> /private/var/folders/20/0qr8ky2558d47l3llvpglfz00000gn/T/pip-install-k3xzvyzv/numpy_e1d80e0ac53a4ad0b7fecb1268e59842/build/temp.macosx-10.9-universal2-3.11/ccompiler_opt_cache_clib.py
            [end of output]
      
        note: This error originates from a subprocess, and is likely not a problem with pip.
        ERROR: Failed building wheel for numpy
      Failed to build numpy
      ERROR: Could not build wheels for numpy, which is required to install pyproject.toml-based projects
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× pip subprocess to install build dependencies did not run successfully.
│ exit code: 1
╰─> See above for output.`

### Environment

```shell
pip install TTS
```

**Comment by Cool-and-Deadly** [comment]
I have the same error when running `pip3 install TTS` on MacOS M2
